documentation/library_guide/macros.rst

```diff
@@ -63,12 +63,35 @@ Macro                              Description
                                    algorithms in the C++ standard libraries of GCC and LLVM.
 ---------------------------------- ------------------------------
 ``ONEDPL_USE_TBB_BACKEND``         This macro controls the use of |onetbb_long| or
-                                   |tbb_long| for parallel policies.
-                                   When the macro is set to 0, algorithms with the ``par`` and ``par_unseq`` policies are only
-                                   executed by the calling thread. This is recommended for code that should not depend on the
-                                   presence of the |onetbb_short| or |tbb_short| library. When the macro is not defined (by default)
-                                   or evaluates to a non-zero value,
-                                   parallel policies are executed using the |onetbb_short| or |tbb_short| library.
+                                   |tbb_long| for parallel policies (``par`` and ``par_unseq``)
+                                   When the macro is set to 0, algorithms with parallel policies do not
+                                   use this backend and try to find the "next" backend according to the priority.
+                                   This is recommended for code that should not depend on the
+                                   presence of the |onetbb_short| or |tbb_short| library.
+                                   When the macro evaluates to a non-zero value, algorithms with parallel policies
+                                   are executed using the |onetbb_short| or |tbb_short| library.
+                                   When the macro is not defined (by default) algorithms with parallel policies
+                                   are executed using the |onetbb_short| or |tbb_short| library unless another
+                                   parallel backend is explicitly enabled.
+
+                                   *TBB backend* has higher priority than *OpenMP backend*
+
+                                   Please note that if all parallel backend macros are set to 0, algorithms with
+                                   parallel policies are executed by the calling thread.
```

Comment:
I'd rewrite the description as follows:

This macro controls the use of |onetbb_long| or |tbb_long| for parallel execution policies (``par`` and ``par_unseq``). When the macro evaluates to a non-zero value, or when it is not defined (by default) and no other parallel backends are explicitly chosen, algorithms with parallel policies are executed using the |onetbb_short| or |tbb_short| library. Setting the macro to 0 prevents use of TBB API for parallel execution and is recommended for code that should not depend on the presence of the |onetbb_short| or |tbb_short| library. If all parallel backends are disabled by setting respective macros to 0, algorithms with parallel policies are executed sequentially by the calling thread.

Reply:
done